components/Navbar/HelpLink.tsx

```diff
@@ -9,8 +9,10 @@ export default function HelpLink({ params }: { params: any }) {
   const t = useTranslations(lang)
 
   const { slug, lesson: lessonId } = params
-  const { theme = chapters[slug].metadata.theme } =
-    lessons[slug][lessonId]?.metadata ?? 'bg-back'
+  const theme =
+    lessons[slug]?.[lessonId]?.metadata.theme ??
+    chapters[slug]?.metadata.theme ??
+    'bg-back'
```

Comment:
Lets put a comment here "If theme was specified on lesson it should take priority over a theme that was specified on a chapter, otherwise fallback to bg-back" or something similar.